dev/com.ibm.ws.request.probe.audit.rest/src/com/ibm/ws/request/probe/audit/rest/service/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+/**
+ * @version 1.0.0
+ */
+@org.osgi.annotation.versioning.Version("1.0.0")
+@TraceOptions(traceGroup = "requestProbe", messageBundle = "com.ibm.ws.request.probe.internal.resources.LoggingMessages")
```

Comment:
```suggestion
@TraceOptions(traceGroup = "requestProbe")
```
No messages file for this bundle.